.vscode/settings.json

```diff
@@ -8,6 +8,7 @@
         "**/node_modules": true,
         "**/pnpm-lock.yaml": true,
     },
+    "files.trimTrailingWhitespace": false,
```

Comment:
VSCode defaults to trailing whitespace (as least on my machine), and it was causing a few whitespace characters to be removed from lines I didn't intend to touch.

For example:
https://github.com/monarchwadia/ragged/blob/main/ragged/src/chat/Chat.ts#L326

I'm not sure if this is your intention or not, so I've added this to the vscode settings here. But of course, I can remove that line and commit the change removing the whitespace.